libmattermost.c

```diff
@@ -2226,6 +2226,34 @@ mm_process_attachment(JsonObject *attachment)
 	return message;
 }
 
+static gint64
+mm_get_channel_approximate_view_time(MattermostAccount *ma, gchar *id)
+{
+	gchar *tmptime = NULL;
+
+	PurpleChat *chat = mm_purple_blist_find_chat(ma, id);
+	if (chat) {
+		tmptime = g_strdup(purple_blist_node_get_string(PURPLE_BLIST_NODE(chat),"channel_approximate_view_time"));

```

Comment:
```suggestion
		tmptime = g_strdup(purple_blist_node_get_string(PURPLE_BLIST_NODE(chat), "channel_approximate_view_time"));

```

Reply:
this was fixed in prev PR review fix, I guess the sync didn't go well and this is reverted, fixed it again